apps/generator/test/test-templates/react-template/__transpiled/test-file.md.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 require('source-map-support/register');
-const generatorReactSdk = require('@asyncapi/generator-react-sdk');
-const jsxRuntime = require('react/cjs/react-jsx-runtime.production.min');
+var generatorReactSdk = require('@asyncapi/generator-react-sdk');
+var jsxRuntime = require('C:/Users/shash/Desktop/generator/node_modules/react/cjs/react-jsx-runtime.production.min.js');
```

Comment:
Here the file path is compatible to windows OS, if someone is using Linux or Mac. Won't it fail to execute?